Misc/NEWS.d/next/Build/2020-02-28-23-51-27.bpo-38380.TpOBCj.rst

```diff
@@ -0,0 +1 @@
+Update Windows builds to use SQLite 3.31.1
```

Comment:
Move this to the `Windows` section (see [previous entries](https://docs.python.org/3/whatsnew/changelog.html#id73))

Reply:
Just a straight `git mv` is fine, there's nothing encoded in the filename/contents that'll matter.

Reply:
Fixed in commit 6a89f6489bc9bfd3c8c31d677f4a8c3fc59f5b0e